pages/404.tsx

```diff
@@ -1,10 +1,27 @@
+import {Box} from '@chakra-ui/react'
 import Header from 'components/header'
+import Wistia from 'components/wistia'
 import Layout from 'layouts/main'
 
 export default function Custom404() {
   return (
     <Layout title='Page not found'>
-      <Header>404: Page not found</Header>
+      <Header>
+        Sorry for the inconvenience, but the page you requested was not found 🤔
+        <br />
+        <br /> Use the links above 👆 or enjoy a full demonstration of Conveyal
+        below 👇
```

Comment:
Tone feels a bit informal compared to the rest of the site. Maybe  just...

```suggestion
        The page you requested was not found. 
        <br />
        <br /> Use the links above 👆 or see the latest Conveyal features in the video
        below 👇
```